recipes/phono3py/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "phono3py" %}
+{% set version = "1.18.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6bafcf6e277408e04af9cb59e0c1805947a163d9fc35795bf541c4458a4951a3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - openblas
+    - liblapacke
```

Comment:
This seems incorrect: try: https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

Reply:
thanks for the reminder